[2,"\n"]
[1,"\u001B[1mpandoc \u001B[22m\n  to: html\n  output-file: Experience.html\n  standalone: true\n  title-prefix: Erick Castro\n  section-divs: true\n  html-math-method: mathjax\n  wrap: none\n  default-image-extension: png\n  css:\n    - styles.css\n  toc: true\n  \n\u001B[1mmetadata\u001B[22m\n  document-css: false\n  link-citations: true\n  date-format: long\n  lang: en\n  editor: visual\n  theme: cosmo\n  title: ''\n  \n"]
[1,"Output created: _site/Experience.html\n\n"]
[1,"\u001B[1m\u001B[34mPreparing to preview\u001B[39m\u001B[22m\n"]
[1,"\n\u001B[32mWatching files for changes\u001B[39m\n\u001B[32mBrowse at \u001B[39m\u001B[4m\u001B[32mhttp://localhost:3653/Experience.html\u001B[39m\u001B[24m\n"]
[1,"\u001B[32mGET: /Experience.html\u001B[39m\n"]
[1,"\u001B[91mERROR: YAMLException: duplicated mapping key (_quarto.yml, 44:5)\n43: \u001B[34m    toc: true\u001B[91m\n44: \u001B[34m    theme: sidebar-left\u001B[91m\n       ~~\n45: \u001B[34meditor: visual\u001B[91m\u001B[39m\n"]
[1,"\u001B[91mERROR: YAMLException: bad indentation of a mapping entry (_quarto.yml, 42:13)\n41: \u001B[34m    theme: cosmo\u001B[91m\n42: \u001B[34m      layout: sidebar-left\u001B[91m\n               ~~\n43: \u001B[34m    css: styles.css\u001B[91m\u001B[39m\n"]
[1,"\u001B[91mERROR: YAMLError: bad indentation of a mapping entry at line 42, column 13:\n          layout: sidebar-left\n                ^\n\nYAMLError: bad indentation of a mapping entry at line 42, column 13:\n          layout: sidebar-left\n                ^\n    at generateError (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:10719:12)\n    at throwError (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:10722:11)\n    at readBlockMapping (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:11361:20)\n    at composeNode (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:11528:84)\n    at readBlockMapping (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:11346:17)\n    at composeNode (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:11528:84)\n    at readBlockMapping (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:11346:17)\n    at composeNode (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:11528:84)\n    at readDocument (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:11642:5)\n    at loadDocuments (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:11679:9)\u001B[39m\n\u001B[32mGET: /Experience.html\u001B[39m\n"]
[1,"\u001B[91mERROR: YAMLException: bad indentation of a mapping entry (_quarto.yml, 42:15)\n41: \u001B[34m    theme: cosmo\u001B[91m\n42: \u001B[34m        layout: sidebar-left\u001B[91m\n                 ~~\n43: \u001B[34m    css: styles.css\u001B[91m\u001B[39m\n"]
[1,"\u001B[91mERROR: YAMLError: bad indentation of a mapping entry at line 42, column 15:\n            layout: sidebar-left\n                  ^\n\nYAMLError: bad indentation of a mapping entry at line 42, column 15:\n            layout: sidebar-left\n                  ^\n    at generateError (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:10719:12)\n    at throwError (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:10722:11)\n    at readBlockMapping (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:11361:20)\n    at composeNode (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:11528:84)\n    at readBlockMapping (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:11346:17)\n    at composeNode (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:11528:84)\n    at readBlockMapping (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:11346:17)\n    at composeNode (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:11528:84)\n    at readDocument (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:11642:5)\n    at loadDocuments (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:11679:9)\u001B[39m\n\u001B[32mGET: /Experience.html\u001B[39m\n"]
[1,"\u001B[91mERROR: YAMLException: bad indentation of a mapping entry (_quarto.yml, 32:7)\n31: \u001B[34m        layout: sidebar-left\u001B[91m\n32: \u001B[34m      - section: \"Menu\"\u001B[91m\n         ~~\n33: \u001B[34m        contents: \u001B[91m\u001B[39m\n"]
[1,"\u001B[91mERROR: YAMLError: bad indentation of a mapping entry at line 32, column 7:\n          - section: \"Menu\"\n          ^\n\nYAMLError: bad indentation of a mapping entry at line 32, column 7:\n          - section: \"Menu\"\n          ^\n    at generateError (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:10719:12)\n    at throwError (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:10722:11)\n    at readBlockMapping (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:11361:20)\n    at composeNode (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:11528:84)\n    at readBlockMapping (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:11346:17)\n    at composeNode (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:11528:84)\n    at readBlockMapping (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:11346:17)\n    at composeNode (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:11528:84)\n    at readDocument (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:11642:5)\n    at loadDocuments (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:11679:9)\u001B[39m\n\u001B[32mGET: /Experience.html\u001B[39m\n"]
[1,"\u001B[32mGET: /Experience.html\u001B[39m\n"]
[1,"\u001B[91mERROR: Project _quarto.yml validation failed.\n\nIn file _quarto.yml\n(line 21, columns 3--8) property name \u001B[34mtheme\u001B[91m is invalid\n20: \u001B[34m        file: Current_research.qmd\u001B[91m\n21: \u001B[34m  theme:\u001B[91m\n     ~~~~~~\n22: \u001B[34m  fonts:\u001B[91m\n\u001B[34mℹ\u001B[91m The error happened in location \u001B[34mwebsite\u001B[91m:\u001B[34mtheme\u001B[91m.\n\n\nIn file _quarto.yml\n(line 22, columns 3--8) property name \u001B[34mfonts\u001B[91m is invalid\n21: \u001B[34m  theme:\u001B[91m\n22: \u001B[34m  fonts:\u001B[91m\n     ~~~~~~\n23: \u001B[34m    navbar:\u001B[91m\n\u001B[34mℹ\u001B[91m The error happened in location \u001B[34mwebsite\u001B[91m:\u001B[34mfonts\u001B[91m.\n\n\nError: Project _quarto.yml validation failed.\n\nIn file _quarto.yml\n(line 21, columns 3--8) property name \u001B[34mtheme\u001B[91m is invalid\n20: \u001B[34m        file: Current_research.qmd\u001B[91m\n21: \u001B[34m  theme:\u001B[91m\n     ~~~~~~\n22: \u001B[34m  fonts:\u001B[91m\n\u001B[34mℹ\u001B[91m The error happened in location \u001B[34mwebsite\u001B[91m:\u001B[34mtheme\u001B[91m.\n\n\nIn file _quarto.yml\n(line 22, columns 3--8) property name \u001B[34mfonts\u001B[91m is invalid\n21: \u001B[34m  theme:\u001B[91m\n22: \u001B[34m  fonts:\u001B[91m\n     ~~~~~~\n23: \u001B[34m    navbar:\u001B[91m\n\u001B[34mℹ\u001B[91m The error happened in location \u001B[34mwebsite\u001B[91m:\u001B[34mfonts\u001B[91m.\n\n    at readAndValidateYamlFromFile (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:42120:15)\n    at async file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:82871:26\n    at async projectContext (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:82723:26)\n    at async refreshProjectConfig (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102808:19)\n    at async file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102928:17\u001B[39m\n"]
[1,"\u001B[91mERROR: Project _quarto.yml validation failed.\n\nIn file _quarto.yml\n(line 21, columns 3--8) property name \u001B[34mtheme\u001B[91m is invalid\n20: \u001B[34m        file: Current_research.qmd\u001B[91m\n21: \u001B[34m  theme:\u001B[91m\n     ~~~~~~\n22: \u001B[34m    fonts:\u001B[91m\n\u001B[34mℹ\u001B[91m The error happened in location \u001B[34mwebsite\u001B[91m:\u001B[34mtheme\u001B[91m.\n\n\nError: Project _quarto.yml validation failed.\n\nIn file _quarto.yml\n(line 21, columns 3--8) property name \u001B[34mtheme\u001B[91m is invalid\n20: \u001B[34m        file: Current_research.qmd\u001B[91m\n21: \u001B[34m  theme:\u001B[91m\n     ~~~~~~\n22: \u001B[34m    fonts:\u001B[91m\n\u001B[34mℹ\u001B[91m The error happened in location \u001B[34mwebsite\u001B[91m:\u001B[34mtheme\u001B[91m.\n\n    at readAndValidateYamlFromFile (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:42120:15)\n    at async file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:82871:26\n    at async projectContext (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:82723:26)\n    at async refreshProjectConfig (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102808:19)\n    at async file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102928:17\u001B[39m\n"]
[1,"\u001B[32mGET: /Experience.html\u001B[39m\n"]
[1,"\u001B[32mGET: /Experience.html\u001B[39m\n"]
[1,"\u001B[32mGET: /Experience.html\u001B[39m\n"]
[1,"\u001B[32mGET: /Experience.html\u001B[39m\n"]
[1,"\u001B[32mGET: /Experience.html\u001B[39m\n"]
[1,"\u001B[32mGET: /Experience.html\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[91mERROR: Project _quarto.yml validation failed.\n\nIn file _quarto.yml\n(line 45, columns 15--17) Field \"\u001B[34mfontsize\u001B[91m\" has value \u001B[34m16\u001B[91m, which must instead be a string\n44: \u001B[34m    toc: true\u001B[91m\n45: \u001B[34m    fontsize: 16\u001B[91m\n                 ~~~\n46: \u001B[34meditor: visual\u001B[91m\n\u001B[31m✖\u001B[91m The value \u001B[34m16\u001B[91m is of type number.\n\u001B[34mℹ\u001B[91m The error happened in location \u001B[34mformat\u001B[91m:\u001B[34mhtml\u001B[91m:\u001B[34mfontsize\u001B[91m.\n\n\nIn file _quarto.yml\n(line 45, columns 15--17) Field \"\u001B[34mfontsize\u001B[91m\" has value \u001B[34m16\u001B[91m, which must instead be a string\n44: \u001B[34m    toc: true\u001B[91m\n45: \u001B[34m    fontsize: 16\u001B[91m\n                 ~~~\n46: \u001B[34meditor: visual\u001B[91m\n\u001B[31m✖\u001B[91m The value \u001B[34m16\u001B[91m is of type number.\n\u001B[34mℹ\u001B[91m The error happened in location \u001B[34mformat\u001B[91m:\u001B[34mhtml\u001B[91m:\u001B[34mfontsize\u001B[91m.\n\n\nError: Project _quarto.yml validation failed.\n\nIn file _quarto.yml\n(line 45, columns 15--17) Field \"\u001B[34mfontsize\u001B[91m\" has value \u001B[34m16\u001B[91m, which must instead be a string\n44: \u001B[34m    toc: true\u001B[91m\n45: \u001B[34m    fontsize: 16\u001B[91m\n                 ~~~\n46: \u001B[34meditor: visual\u001B[91m\n\u001B[31m✖\u001B[91m The value \u001B[34m16\u001B[91m is of type number.\n\u001B[34mℹ\u001B[91m The error happened in location \u001B[34mformat\u001B[91m:\u001B[34mhtml\u001B[91m:\u001B[34mfontsize\u001B[91m.\n\n\nIn file _quarto.yml\n(line 45, columns 15--17) Field \"\u001B[34mfontsize\u001B[91m\" has value \u001B[34m16\u001B[91m, which must instead be a string\n44: \u001B[34m    toc: true\u001B[91m\n45: \u001B[34m    fontsize: 16\u001B[91m\n                 ~~~\n46: \u001B[34meditor: visual\u001B[91m\n\u001B[31m✖\u001B[91m The value \u001B[34m16\u001B[91m is of type number.\n\u001B[34mℹ\u001B[91m The error happened in location \u001B[34mformat\u001B[91m:\u001B[34mhtml\u001B[91m:\u001B[34mfontsize\u001B[91m.\n\n    at readAndValidateYamlFromFile (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:42120:15)\n    at async file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:82871:26\n    at async projectContext (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:82723:26)\n    at async refreshProjectConfig (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102808:19)\n    at async file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102928:17\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /publications.html\u001B[39m\n"]
[1,"\u001B[32mGET: /Experience.html\u001B[39m\n"]
[1,"\u001B[32mGET: /Res_experience.html\u001B[39m\n"]
[1,"\u001B[32mGET: /skills.html\u001B[39m\n"]
[1,"\u001B[32mGET: /Current_research.html\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /Current_research.html\u001B[39m\n"]
[1,"\u001B[32mGET: /Res_experience.html\u001B[39m\n"]
[1,"\u001B[32mGET: /Experience.html\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /Res_experience.html\u001B[39m\n"]
[1,"\u001B[32mGET: /skills.html\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /publications.html\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /publications.html\u001B[39m\n"]
[1,"\u001B[91mERROR: Too many open files (os error 24)\n\nError: Too many open files (os error 24)\n    at new FsWatcher (deno:runtime/js/40_fs_events.js:19:23)\n    at Object.watchFs (deno:runtime/js/40_fs_events.js:64:12)\n    at runWatcher (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102957:30)\n    at file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102968:45\n    at Array.forEach (<anonymous>)\n    at watchForChanges (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102962:37)\u001B[39m\n\u001B[32mGET: /Experience.html\u001B[39m\n"]
[1,"\u001B[91mERROR: Too many open files (os error 24)\n\nError: Too many open files (os error 24)\n    at new FsWatcher (deno:runtime/js/40_fs_events.js:19:23)\n    at Object.watchFs (deno:runtime/js/40_fs_events.js:64:12)\n    at runWatcher (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102957:30)\n    at file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102968:45\n    at Array.forEach (<anonymous>)\n    at watchForChanges (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102962:37)\u001B[39m\n"]
[1,"\u001B[91mERROR: Too many open files (os error 24)\n\nError: Too many open files (os error 24)\n    at new FsWatcher (deno:runtime/js/40_fs_events.js:19:23)\n    at Object.watchFs (deno:runtime/js/40_fs_events.js:64:12)\n    at runWatcher (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102957:30)\n    at file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102968:45\n    at Array.forEach (<anonymous>)\n    at watchForChanges (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102962:37)\u001B[39m\n\u001B[32mGET: /Res_experience.html\u001B[39m\n"]
[1,"\u001B[91mERROR: Too many open files (os error 24)\n\nError: Too many open files (os error 24)\n    at new FsWatcher (deno:runtime/js/40_fs_events.js:19:23)\n    at Object.watchFs (deno:runtime/js/40_fs_events.js:64:12)\n    at runWatcher (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102957:30)\n    at file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102968:45\n    at Array.forEach (<anonymous>)\n    at watchForChanges (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102962:37)\u001B[39m\n"]
[1,"\u001B[91mERROR: Too many open files (os error 24)\n\nError: Too many open files (os error 24)\n    at new FsWatcher (deno:runtime/js/40_fs_events.js:19:23)\n    at Object.watchFs (deno:runtime/js/40_fs_events.js:64:12)\n    at runWatcher (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102957:30)\n    at file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102968:45\n    at Array.forEach (<anonymous>)\n    at watchForChanges (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102962:37)\u001B[39m\n"]
[1,"\u001B[32mGET: /skills.html\u001B[39m\n"]
[1,"\u001B[91mERROR: Too many open files (os error 24)\n\nError: Too many open files (os error 24)\n    at new FsWatcher (deno:runtime/js/40_fs_events.js:19:23)\n    at Object.watchFs (deno:runtime/js/40_fs_events.js:64:12)\n    at runWatcher (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102957:30)\n    at file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102968:45\n    at Array.forEach (<anonymous>)\n    at watchForChanges (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102962:37)\u001B[39m\n"]
[1,"\u001B[91mERROR: Too many open files (os error 24)\n\nError: Too many open files (os error 24)\n    at new FsWatcher (deno:runtime/js/40_fs_events.js:19:23)\n    at Object.watchFs (deno:runtime/js/40_fs_events.js:64:12)\n    at runWatcher (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102957:30)\n    at file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102968:45\n    at Array.forEach (<anonymous>)\n    at watchForChanges (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102962:37)\u001B[39m\n\u001B[32mGET: /Current_research.html\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[91mERROR: Too many open files (os error 24)\n\nError: Too many open files (os error 24)\n    at new FsWatcher (deno:runtime/js/40_fs_events.js:19:23)\n    at Object.watchFs (deno:runtime/js/40_fs_events.js:64:12)\n    at runWatcher (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102957:30)\n    at file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102968:45\n    at Array.forEach (<anonymous>)\n    at watchForChanges (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102962:37)\u001B[39m\n"]
[1,"\u001B[91mERROR: Too many open files (os error 24)\n\nError: Too many open files (os error 24)\n    at new FsWatcher (deno:runtime/js/40_fs_events.js:19:23)\n    at Object.watchFs (deno:runtime/js/40_fs_events.js:64:12)\n    at runWatcher (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102957:30)\n    at file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102968:45\n    at Array.forEach (<anonymous>)\n    at watchForChanges (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102962:37)\u001B[39m\n\u001B[32mGET: /\u001B[39m\n"]
[1,"\u001B[91mERROR: YAMLException: duplicated mapping key (_quarto.yml, 3:3)\n2: \u001B[34m  type: website\u001B[91m\n3: \u001B[34m  type: website\u001B[91m\n    ~~\n4: \u001B[34m  output-dir: docs\u001B[91m\u001B[39m\n"]
[1,"\u001B[91mERROR: Too many open files (os error 24)\n\nError: Too many open files (os error 24)\n    at new FsWatcher (deno:runtime/js/40_fs_events.js:19:23)\n    at Object.watchFs (deno:runtime/js/40_fs_events.js:64:12)\n    at runWatcher (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102957:30)\n    at file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102968:45\n    at Array.forEach (<anonymous>)\n    at watchForChanges (file:///usr/lib/rstudio/resources/app/bin/quarto/bin/quarto.js:102962:37)\u001B[39m\n"]
